release/dist/synology/files/config

```diff
@@ -0,0 +1,12 @@
+{
+	".url": {
+		"SYNO.SDS.Tailscale": {
+			"type": "url",
+			"version": "1.8.3",
```

Comment:
Does this need to be generated with the current version number?

Reply:
It's a TODO in the code I ported over from tailscale-synology. I decided to not get to the bottom of it in this change, so I left it as-is and ported the TODO to figure it out.